src/Components/NotFoundPage.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const NotFoundPage = (props) => {
+  return (
+    <div>
+      <h2>404 Page not found</h2>
+      <p>Hey looks like we couldn't find the page you were looking for!</p>
+      <p>Please use the menu/nav to find what you need?!?</p>
+    </div>
+  );
+};
+
+export default NotFoundPage;
```

Comment:
The footer is not at the bottom of the page when the user is shown the 404 page. Is there a way we can position the footer to the bottom of the window?

Reply:
No blocks to merging, just an aesthetic preference on my part :)